app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @project.name %></h1>
+<p>Material: <%= @project.material %></p>
+<p>Challenge Theme: <%= @challenge.theme %></p>
+<p>Contestant Count: <%= @project.contestant_count %></p>
+
+<a href="/projects/<%= @project.id %>">View Project</a>
```

Comment:
This button routes the user back to the same page right? Did you want to route it to a different page?

Also, keep in mind we'd like you to start using `link_to` and `button_to` rather than raw HTML links